features/support/api_helpers.rb

```diff
@@ -190,7 +190,7 @@ def create_supplier
   JSON.parse(response.body)['suppliers']
 end
 
-def create_live_service(framework_slug, lot_slug, supplier_id)
+def create_live_service(framework_slug, lot_slug, supplier_id, role=nil)
```

Comment:
Feels like a method called `create_live_service` shouldn't be restricted to three DOS lots only.  I guess we flesh it out with G-Cloud and labs as they are necessary though :/